charts/pwa/README.md

```diff
@@ -55,6 +55,25 @@ Both `cacheIgnoreParams` and `multiChannel` parameters take precedence over any
 | `hybrid.backend.service` | ICM Web Adapter service name                 | `icm-web`     |
 | `hybrid.backend.port`    | ICM Web Adapter service port                 | `443`         |
 
+## Shared Redis Cache
+
+> [!IMPORTANT]
+> The shared Redis cache for the nginx containers requires Intershop PWA version 5.0.0 or newer.
+
+The PWA Helm chart supports a shared Redis cache for the nginx containers. To enable it, add the following configuration to your values file:
+
+```yaml
+redis:
+  uri: rediss://user:password@redis.cloud.com:6379
+  # keepCache: true
+```
+
+Unless `keepCache` is explicitly set to `true`, the cache will be flushed on every deployment using `redis-cli` with a `flushdb` command on the supplied URI.
+
+This chart does not deploy a Redis instance. You must provide one yourself. We recommend using a cloud service.
+
+If you want to deploy a Redis instance yourself, be aware that the PWA implementation does not support Redis Cluster or Redis Sentinel connections.
```

Comment:
@cortlepp-intershop: Do you have an opinion on that sentence?

Reply:
I'm not sure what you mean by that. The pwa-redis is currently required to be a separate instance (AFAIK) from the internal pagecache one. If this is one of their limitations, that may be ^^. For the internal pagecache we don't have that limitation (at least it can be configured, but I have never tried this specific configuration. But if it doesn't work this would be an error in our redis client library).